demystifying_large_language_models/demystifying_large_language_models.md

```diff
@@ -0,0 +1,691 @@
+
+We disagree, however, with the answers to this quiz question, so we'll pose the question to you and explain what ChatGPT suggested the correct answers were and what we think the correct answers are!
+
+------
+
+**Which of these are risks or costs associated with using a large language model?**
+
+[[X]] A) Increased computational resources and energy consumption.
+[[X]] B) Potential biases and perpetuation of societal inequalities.
+[[X]] C) Limited interpretability and transparency of model outputs.
+[[X]] D) Higher risk of data breaches and security vulnerabilities.
+[[X]] E) Reduced creativity and originality in generated content.
+[[X]] F) Decreased accessibility for users with limited internet connectivity.
+[[X]] G) Difficulty in adapting to new language patterns and emerging slang.
+***
+<div class = "answer">
+
+This is what ChatGPT claims:
+
+> Correct answers: <br/>
+> A) Increased computational resources and energy consumption. <br/>
+> B) Potential biases and perpetuation of societal inequalities. <br/>
+
+We, however, see all of these as at least **potential** risks:
+
+* LLMS require enormous energy and computational resources (A)
+* LLMs are built on human language which reflect societal bias, and this will appear in their outputs (B)
+* LLMs are built on complex deep learning models and it is hard to grasp how an LLM generated particular content (C)
+* Because of the risks of anthroporphizing them and disclosing sensitive data when giving prompts, LLMs can introduce new data breach and security risks (D)
+* Perhaps controversially, we believe that human writers are capable of novel uses of language that language models cannot fully mimic (E)
+* It's possible that LLMs will exacerbate the digital divide, reduce employment for automatable jobs, and decrease access to technology for some (F)
+* LLMs are often trained on a static dataset, which, unless it is constantly updated, represents a point in time that will not capture novel language use (G)
+
+</div>
+***
+
+Finally, we're going to leave AI behind for one final question: 
+
+**Think about your own research, and write out how a large language model could improve or accelerate your work.**
+
+Clicking "submit" doesn't actually send this to us, and your answer isn't being evaluated in any way.
+
+[[___ ___ ___]]
+
+<details>
+
+<summary>Click here to see our answer</summary>
+
+<div class = "answer">
+
+We don't know what your research entails, or what your intentions are for the use of AI in your career.  However, we encourage you to:
+
+* Make no assumptions about the accuracy of any text generated by LLMs 
```

Comment:
So the question above is about how to improve/accelerate work, but this answer explanation is about what pitfalls to be careful of? 

If you want to include these reminders to drive it home, that's fine, but I think it would make more sense to first also include some examples that address the question that was asked.

Reply:
Agree, I added additional context here, @franzenr, tell me what you think!

Reply:
hmmm, I'm not seeing any changes to this section? is it possible the changes didn't get pushed? @pm0kjp

Reply:
jk, that was a me problem. Not sure why I wasn't seeing it, but I can see it now! 🤦

Reply:
LGTM :)